src/actions/wods/create.cr

```diff
@@ -0,0 +1,11 @@
+class Wods::Create < BrowserAction
+  action do
+    WodForm.save(params) do |form, wod|
+      if wod
+        redirect to: Home::Index
```

Comment:
If you do the comment above then this could be `Wods::New` or something. A bit more clear I think